packages/site/src/pages/visualTests/VisualTestChipPage.tsx

```diff
@@ -194,6 +194,31 @@ export const VisualTestChipPage = () => {
               </Grid.Cell>
             </Grid>
           </section>
+
+          {/* Disabled State */}
+          <section>
+            <Text size="large">Disabled State</Text>
+            <Grid>
+              <Grid.Cell size={{ xs: 12, md: 6 }}>
+                <Flex
+                  direction="row"
+                  gap="small"
+                  template={["shrink", "shrink"]}
+                >
+                  <Chip label="Disabled Chip" disabled>
+                    <Chip.Prefix>
+                      <Icon name="alert" size="small" />
+                    </Chip.Prefix>
+                  </Chip>
+                  <Chip
+                    label="Disabled with Heading"
+                    heading="Error"
```

Comment:
big ol' nitpick: these aren't the Error state - but I am not going to ask you to regenerate them all again. I know it takes a while, and the heading content doesn't really matter beyond what it looks like